task5/src/main/java/GetServlet.java

```diff
@@ -0,0 +1,16 @@
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/get")
+public class GetServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException{
+        String name = req.getParameter("name");
+        ServletOutputStream stream = resp.getOutputStream();
+        stream.print("Name is " + name);
```

Comment:
What happens if "name" is not actually present?
And what if it exceeds the 100 character limit?